vantiq-operations/docs/eng/mongodb.md

```diff
@@ -0,0 +1,56 @@
+# Introduction to MongoDB configuration on the Vantiq
+
+## About the availability of MongoDB configured on the Vantiq
+
+### TL;DR
+At least two of the three mirrored configuration servers need to be up and running.
+
+### Details
+MongoDBs in the Vantiq are the three-server mirror configuration, and the breakdown is the following.  
+
+- Three servers are up and running: Primary + Secondary + Secondary
+- Two servers are up and running: Primary + Secondary
+- One server is up and running: Secondary
+
+Primary is for write only, Secondary is for read only.  
+If the servers become single due to AZ failure etc., it will be able to read only, so attempting to write will result in a timeout error.  
+
+![timeout_error](../../imgs/mongodb/timeout_error.png)
+
+After a few minutes, the Vantiq pod is unable to write regularly (I guess) and the Container of the Pod itself is no longer `Ready`.  
```

Comment:
After a few minutes, as the Vantiq pod is no longer able to do the regular write access, the container of the pod itself turns into NOT `Ready`